packages/tabs/addon/components/nucleus-tabs/tab-panel.js

```diff
@@ -0,0 +1,101 @@
+import Component from '@ember/component';
+import { classNames, attributeBindings, classNameBindings, tagName, layout as templateLayout } from '@ember-decorators/component';
+import layout from '../../templates/components/nucleus-tabs/tab-panel';
+import { get, computed } from '@ember/object';
+import defaultProp from '@freshworks/core/utils/default-decorator';
+import { once } from '@ember/runloop';
+
+/**
+  __Usage:__
+
+  [Refer component page](/docs/components/nucleus-tabs)
+
+  @class Nucleus Tab Panel
+  @namespace Components
+  @extends Ember.Component
+  @public
+*/
+@tagName('div')
+@templateLayout(layout)
+@classNames('nucleus-tabs__panel')
+@classNameBindings('isActive:active')
+@attributeBindings('tabindex')
+@attributeBindings('role')
+@attributeBindings('aria-labelledby')
+class TabPanel extends Component {
+
+  /**
+  * disabled
+  *
+  * @field disabled
+  * @type string
+  * @default 'false'
+  * @readonly
+  * @public
+  */
+  @defaultProp
+  disabled = 'false';
+
+  /**
+  * tabindex
+  *
+  * @field tabindex
+  * @default '0'
+  * @type string
+  * @public
+  */
+  tabindex = '0';
+
+  /**
+  * role
+  *
+  * @field role
+  * @type string
+  * @public
+  */
+  role = 'tabpanel'
+
+  /**
+  * isActive
+  *
+  * @field isActive
+  * @type boolean
+  * @public
+  */
+  @computed('props.[]', function() {
```

Comment:
Instead of computing the values inside `props`, we could directly pass the prop elements from the parent component and consume accordingly

Reply:
Done. Now directly passing the values.